examples/complete_workflow/ClientCredentialsCompleteWorkFlow.java

```diff
@@ -0,0 +1,62 @@
+package complete_workflow;
+
+import java.io.IOException;
+
+import org.apache.hc.core5.http.ParseException;
+
+import com.wrapper.spotify.SpotifyApi;
+import com.wrapper.spotify.exceptions.SpotifyWebApiException;
+import com.wrapper.spotify.model_objects.credentials.ClientCredentials;
+import com.wrapper.spotify.model_objects.specification.Album;
+import com.wrapper.spotify.model_objects.specification.ArtistSimplified;
+import com.wrapper.spotify.requests.authorization.client_credentials.ClientCredentialsRequest;
+import com.wrapper.spotify.requests.data.albums.GetAlbumRequest;
+import com.wrapper.spotify.requests.data.search.simplified.SearchArtistsRequest;
+
+public class ClientCredentialsCompleteWorkFlow {
+	
+	/*
+	 * With this method we simply get our clientId and clientSecret from the Spotify Developer DashBoard after registering our app. 
+	 * After copying it here we will add them to our spotifyApi. With this we can search for information that is not related to a 
```

Comment:
Please add a comma character before of after "here", depending on what wording you intended.